server/scripts/2021TaxSeasonDataScript.php

```diff
@@ -0,0 +1,420 @@
+			$secondAppointmentTimeId = insertAppointmentTime("$date 18:00:00", 6, 6, $siteId);
+		}
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting F Street Community Center data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertSoutheastCommunityCollegeData() {	
+	GLOBAL $DB_CONN;
+
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Southeast Community College data has already been inserted');
+	}
+ 	
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = 10; // Manually obtained from PROD DB
+
+		// Mondays, Feb 1 - Mar 1
+		$mondayDates = getWeeklyDatesFromRange('2021-02-01', '2021-03-01');
+		foreach ($mondayDates as $date) {
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 12:00:00", 4.5, 6, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 4.5, 6, $siteId);
+		}
+
+		// Thursdays, Feb 4 - mar 25 or apr 8
+		$thursdayDates = getWeeklyDatesFromRange('2021-02-04', '2021-04-08');
+		foreach ($thursdayDates as $date) {
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 4.5, 6, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 16:00:00", 4.5, 6, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 4.5, 6, $siteId);
+		}
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Southeast Community College data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertNebraskaUnionData() {
+	GLOBAL $DB_CONN;
+	
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Nebraska Union data has already been inserted');
+	}
+
+	$siteId = 11; // Manually obtained from PROD DB
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		// Saturdays
+		$saturdayDates = getWeeklyDatesFromRange('2021-01-30', '2021-03-06');
+		foreach ($saturdayDates as $date) {
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 10:00:00", 7.5, 6, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 11:00:00", 7.5, 6, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 12:00:00", 7.5, 6, $siteId);
+			$fourthAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 7.5, 6, $siteId);
+			$fifthAppointmentTimeId = insertAppointmentTime("$date 14:00:00", 7.5, 6, $siteId);
+			$sixthAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 7.5, 6, $siteId);
+		}
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Nebraska Union data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertAsianCommunityAndCulturalCenterData() {
+	GLOBAL $DB_CONN;
+	
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Asian Community and Cultural Center data has already been inserted');
+	}
+
+	$siteId = insertSite("Asian Community And Cultural Center", "", "");
```

Comment:
I thought this "Asian Community and Cultural Center" site existed already--does it not?

Reply:
Not that I see! Thanks for double checking.